CRM/ManualDirectDebit/Queue/Task/BatchSubmission/PaymentItem.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+class CRM_ManualDirectDebit_Queue_Task_BatchSubmission_PaymentItem {
```

Comment:
This class and `CRM_ManualDirectDebit_Queue_Task_BatchSubmission_InstructionItem` are also very similar, which speaks to me of an opportunity to refactor into a parent class and leave on each concrete class just what differs. If you think it could help. Otherwise I think it's fine, as they're such small classes, might not be worth the hassle.

Reply:
I think I will keep this simple and leave it as